jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/dao/GenericJdbcDatabaseAccessor.java

```diff
@@ -364,6 +366,16 @@ protected Properties getConnectionPoolProperties(Configuration conf) throws Exce
       passwd = Utilities.getPasswdFromKeystore(keystore, key);
     }
 
+    if (passwd == null) {
```

Comment:
same here. the password store is last in the order of attempting to find a password. Should this be reversed? if it is reversed, are there any backward compatibility issues? (I dont think there are but just asking)